crates/runtime/src/metrics.rs

```diff
@@ -269,7 +269,7 @@ pub(crate) mod telemetry {
         TELEMETRY_METER
             .u64_counter("query_processed_bytes")
             .with_description("Number of bytes processed by the runtime.")
-            .with_unit("bytes")
+            .with_unit("By")
```

Comment:
This is unrelated change, added for consistency and to match proposal to follow UCUM units